.github/workflows/analyze-project.yaml

```diff
@@ -0,0 +1,39 @@
+name: Analyze Project
+on:
+  workflow_call:
+jobs:
+  windows-analysis:
```

Comment:
I think "windows" should suffice in this case as the "analysis" part is provided by the workflow name.

If it has to be part of the job name, then I'd prefer "analysis-windows", as it fits better with "analysis-linux" and "analysis-macros" later.